docs/io/unified.rst

```diff
@@ -202,6 +202,7 @@ ascii.fixed_width_no_header    Yes          :class:`~astropy.io.ascii.FixedWidth
                  ascii.ipac    Yes          :class:`~astropy.io.ascii.Ipac`: IPAC format table
                 ascii.latex    Yes    .tex  :class:`~astropy.io.ascii.Latex`: LaTeX table
             ascii.no_header    Yes          :class:`~astropy.io.ascii.NoHeader`: Basic table with no headers
+                  ascii.qdp    Yes    .qdp   QDP_: Quick and Dandy Plotter files
```

Comment:
Should be a `:class: ..` like the others where the class docstring has the reference info.

Reply:
Also include this new format in `astropy/docs/io/ascii/index.html`  around line 260.

Reply:
Done